source/python/README.md

```diff
@@ -2453,7 +2456,7 @@ Target: EQJ = equatorial system, using equator at the J2000 epoch.
 | [`Observer`](#Observer) | `observer` | A location near the Earth's mean sea level that define's the observer's horizon. |
 
 ### Returns: [`RotationMatrix`](#RotationMatrix)
-A rotation matrix that converts HOR to EQJ at `time` and for `observer`.
+A rotation matrix that converts HOR to EQD at `time` and for `observer`.
```

Comment:
I know I recently fixed this "EQD" that is supposed to say "EQJ", so I am not sure how it came back in your changes.